CHANGELOG.md

```
@@ -1,8 +1,43 @@
+# [0.4.0-beta.1](https://github.com/AntoniadisCorp/deepscrape/compare/v0.3.0...v0.4.0-beta.1) (2025-11-17)
+
+
+### Bug Fixes
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)